dev/pages/Attention.vue

```diff
@@ -108,7 +106,6 @@ const highlightShowing = ref(false)
               right
               v-model="calloutShowing"
               class="ml-8"
-              :target-el="calloutTarget ? calloutTarget.$el : null"
```

Comment:
Normally, we'd need target element reference for the Attention component to figure out where it should be positioned towards that target (see `recompute` function). But now I see that this does not apply to Attention of callout variant, because it has relative position, unlike other variants which are absolutely positioned 🙌